nat-lab/tests/conftest.py

```diff
@@ -232,7 +233,7 @@ async def _copy_vm_binaries_if_needed(items):
                 return
 
 
-def save_dmesg_from_host():
+def save_dmesg_from_host(prefix):
```

Comment:
Nit: Wouldn't this technically be a suffix, since it's `dmesg-{prefix}` rather than `{prefix}-dmesg`?

Reply:
you are right... mixed up the words :laughing: will fix it in a second

Reply:
fixed